test/util/nightwatch-commands/axeCheck.js

```diff
@@ -29,11 +29,6 @@ export function command(context, config, _callback) {
       done({ err, results });
     });
   }, [context, (config || {}).rules || this.globals.rules || ['section508', 'wcag2a']], response => {
-    if (response.state !== 'success') {
```

Comment:
It's just what I encountered while running it. I haven't catalogued which browsers respond in which ways, but I could put that together pretty easily. The thing that struck me here was that this test is really just a short circuit for things that are tested with more depth below. If the response is totally bogus, it's going to fail one of those other tests, and if it's well-formed but has violations then it's going to be flagged as well.